tasks/install.yml

```diff
@@ -31,19 +31,19 @@
 
 - name: Decompress the binary
   become: false
-  shell: "bzip2 -dc /tmp/restic_{{ restic_version }}_{{ ansible_system | lower }}_{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}.bz2 > /tmp/restic"
+  shell: "bzip2 -dc /tmp/restic_{{ restic_version }}_{{ ansible_system | lower }}_{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}.bz2 > /tmp/restic_{{ restic_version }}"
   args:
-    creates: "/tmp/restic"
+    creates: "/tmp/restic_{{ restic_version }}"
   delegate_to: localhost
   run_once: true
   check_mode: false
 
 - name: Propagate restic binary
   copy:
-    src: /tmp/restic
+    src: "/tmp/restic_{{ restic_version }}"
     dest: '{{ restic_install_path }}/restic'
     mode: '0750'
-    owner: 'root'
+    owner: '{{ restic_user }}'
```

Comment:
Using root as a binary owner is a security feature.
Intention behind using `root` as an owner and setting mode to `0750` was that a user running restic command would be able to run it when he is in the `restic_group` group, but he won't be allowed to modify it.

Reply:
ah, my bad! I somehow misread the mode and thought 5 was r+w whereas it's r+x -- it's much better how it was, I will ditch the second commit then! :)